man/coqnative.1

```diff
@@ -0,0 +1,80 @@
+.TH COQ 1
+
+.SH NAME
+coqnative \- The Coq native compiler
+
+
+.SH SYNOPSIS
+.B coqnative
+[
+.B options
+]
+.I filename
+
+
+.SH DESCRIPTION
+
+.B coqnative
+compiles vo files to binary files used by the native compute mechanism
+of the Coq Proof Assistant. See the Reference Manual for more
+information. It returns with exit code 0 if all the requested tasks
+succeeded. A non-zero return code means that something went wrong.
```

Comment:
Maybe should be "the Coqnative Reference Manual" and perhaps could include a hyperlink (if that's commonly done in man pages).

The two sentences about exit codes are not needed.  They are standard behavior for commands.